config/routes.config.js

```diff
@@ -13,6 +13,7 @@ const routes = [
   { name: 'question-ccb', path: { en: '/CCB', fr: '/ARC' } },
   { name: 'question-student-debt', path: { en: '/student-debt', fr: '/dette-des-etudiants' } },
 
+  { name: 'prepare', path: { en: '/prepare', fr: '/prepare' } },
```

Comment:
French route should have an r at the end. 

https://translate.google.com/#view=home&op=translate&sl=en&tl=fr&text=Prepare
https://www.deepl.com/en/translator#en/fr/Prepare

Reply:
frenchified